docs/exts/eliot_metrics.py

```diff
@@ -0,0 +1,109 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is kind of nuts. I'm inclined to fold this proof-of-concept into Markus, but I want to bounce it off some people, first.

The Markus issue is https://github.com/willkg/markus/issues/15 .